plugins/pmsmart/src/test/java/org/itsallcode/whiterabbit/plugin/pmsmart/integration/ConfigMock.java

```diff
@@ -0,0 +1,69 @@
+package org.itsallcode.whiterabbit.plugin.pmsmart.integration;
+
+import java.nio.file.Path;
+import java.time.Duration;
+import java.util.Locale;
+import java.util.Optional;
+
+import org.itsallcode.whiterabbit.logic.Config;
+
+public class ConfigMock implements Config
```

Comment:
You can also use Mockito for creating mocks. Then you don't need a separate class.

Reply:
Replaced by local static final class and narrow interfaces.